docs/detections/rules-ui-create.asciidoc

```diff
@@ -29,20 +29,31 @@ generated for each value.
 For example, if the threshold `field` is `source.ip` and its `value` is `10`, an
 alert is generated for every source IP address that appears in at least 10 of
 the rule's search results.
+* *Event correlation*: Searches the defined indices and creates an alert when results match an
```

Comment:
Is there any way we could link add a link here to the section where we walk through details on how to create a rule of this type? Essentially a short cut. If the user wants to create an Event Correlation rule they could click on this link and it would take them to the rule creation details and example.